### DISCUSSION TITLE
Transaction grouping

### DISCUSSION THREAD

**Discussion by icgncl** [getsentry/sentry-python#2416] [Q&A] [answered]
Hi everyone, I am currently utilizing the Sentry SDK for Python. However, I've encountered an issue where it incorrectly groups two endpoints when they share two parameters in the URL. For example, I have two endpoints as follows:

/api/order/{id}
/api/order/{id}/product/{pid}
Unfortunately, in the Sentry transaction data, these endpoints are consolidated into a single endpoint, which is /api/order/{id}. This behavior is not as expected. 

Sentry-sdk version =1.31.0

**Comment by smeubank** [comment]
hi @icgncl, 

could you provide more info about your app? Are you using django rest framework, fastapi etc?

and you SDK init so we see what integrations you are using

**Comment by icgncl** [comment]
hi @smeubank 

Here it is : 
```   
sentry_sdk.init(
        dsn=BACKEND_SENTRY_DSN,
        integrations=[DjangoIntegration(), CeleryIntegration(monitor_beat_tasks=True)],
        send_default_pii=False,
        release=COMMIT_HASH_FULL,

        environment = ENV_NAME,

        traces_sampler=traces_sampler,
    )
```

**Comment by icgncl** [comment]
In sentry-sdk==1.5.10, there is no issue with grouping however with this version (1.5.10), I cannot use CeleryIntegration

**Reply by smeubank** [reply]
@icgncl what do you mean you cannot use the CeleryIntegration with this version? You set it and it doesn't work or there is some other issue?

hmmm @antonpirker do you know why the CeleryIntegration doesn't work with this version

**Reply by icgncl** [reply]
@smeubank In version 1.5.10, the 'monitor_beat_tasks' feature is not available, which prevents me from using Celery Integration. However, in version 1.31.0, I can utilize Celery Integration.

As I mentioned earlier, the new version (1.31.0) consolidates different endpoints into a single endpoint so I cannot go with new version

**Reply by smeubank** [reply]
ah ah sorry i was misreading the semver 

i see ok in later version the parameterization of these routes is off and you also want to use CeleryBeats we will investigate. This might need to be converted to an issue, since this might be a regression

**Reply by smeubank** [reply, marked as answer]
https://github.com/getsentry/sentry-python/issues/2392

this is regression. I would propose to close this discussion and I will comment and link in that issue, then try to see if we can get it addressed next week.
